### ISSUE TITLE
Some tickets are not being automatically deleted by mongo TTL

### ISSUE THREAD

**Issue by kofalt** [flywheel-io/core#1428]
Turns out, if you create a TTL on a key that is never used, mongo happily allows the docs to exist forever.

At minimum, job tickets are sticking around forever, one site has ~7000 of them.

1. [ ] Add documentation to the TTL index setup, so that we don't make this mistake again.
1. [ ] For each index, confirm which have this problem.
1. [ ] Write a database upgrade to add this field to the afflicted collections (now? epoch?) or delete.
1. [ ] Fix whichever tickets to have `timestamp`, or `modified`...

**Comment by ryansanford** [comment]
@nagem 
One customer has 12 jobs that lingered in a running state for 2 weeks because the `/reap` endpoint doesn't process jobs that have a ticket. We are remediating by removing the offending job_ticket records directly in the DB.

**Comment by nagem** [comment]
@ryansanford I assume there was an unrelated problem causing them to need to be reaped, correct?

**Comment by ryansanford** [comment]
@nagem Correct. Root cause of that unrelated problem is still at large, and for now further investigation can wait until the site is on the latest release.
